nixos/tests/nix-store-gcs-proxy.nix

```diff
@@ -0,0 +1,29 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+{
+  name = "nix-store-gcs-proxy";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ mrkkrp ];
+  };
+
+  machine = { pkgs, ... }: {
+    services.nix-store-gcs-proxy = {
+      foo = {
+        address = "localhost:3333";
+      };
+      bar = {
+        address = "localhost:5555";
+      };
+    };
+  };
+
+  testScript = ''
+    start_all()
+    # the nix-store-gcs-proxy executable needs valid credentials to be
```

Comment:
Then this test is basically useless? It doesn't even wait for the VM to start. It just starts it and then terminates.

cc @zimbatm Since you authored (most of?) the proxy: Is there something sensible we can actually test here? I would really like the daemon come up and respond to http requests.

Reply:
I guess we could add a `/_health` end-point like this: https://github.com/tweag/nix-store-gcs-proxy/pull/7 ? Ideally the _health endpoint also checks that it has access to the bucket but in this context it would be hard to emulate that.

Reply:
Let's just drop this test as it's not really useful in its current form.

Reply:
It could be emulated using something like that https://github.com/fsouza/fake-gcs-server.
For the time being this might be too complicated...

Reply:
Yeah, I think we have been setting the bar of acceptance too high and it's unfair to @mrkkrp